Report.md

```diff
@@ -912,8 +912,8 @@ extra knowledge of the array to dynamically determine how much data to send.
 Each multi-process merge involves both processes sending each other the same
 quantity of information every time.
 
-![alt text](merge_ipynb/Merge_Plots/Mergesort%20Strong%20Scaling%20(comm,%20n=268435456).png)
-![alt text](merge_ipynb/Merge_Plots/Mergesort%20Strong%20Scaling%20Speedup%20(comm,%20Random).png)
+![alt text](radix_ipynb/Radix_Plots/Radix%20Strong%20Scaling%20(comm,%20n=268435456).png)
+![alt text](radix_ipynb/Radix_Plots/Radix%20Strong%20Scaling%20Speedup%20(comm,%20Random).png)
```

Comment:
This is part of the Merge Sort Communication section. These shouldn't be Radix Sort graphs.